project/src/pages/room/room.tsx

```diff
@@ -54,120 +49,66 @@ function RoomPage({ offers }: RoomPageProps): JSX.Element {
               </div>
               <div className="property__rating rating">
                 <div className="property__stars rating__stars">
-                  <span style={{ width: '80%' }}></span>
+                  <span style={{ width: calculateRatingPercent(rating) }}></span>
                   <span className="visually-hidden">Rating</span>
                 </div>
                 <span className="property__rating-value rating__value">4.8</span>
               </div>
               <ul className="property__features">
                 <li className="property__feature property__feature--entire">
-                  Apartment
+                  {capitalize(type)}
                 </li>
                 <li className="property__feature property__feature--bedrooms">
-                  3 Bedrooms
+                  {`${bedrooms} Bedrooms`}
```

Comment:
в jsx строки не нужны, так тоже можно
```suggestion
                  {bedrooms} Bedrooms
```